martian-lab/tests/error_test/expected/SUM_SQUARES/SUM_SQUARES/fork0/chnk3/_stackvars

```diff
@@ -1,44 +1,46 @@
-   0: anyhow::private::format_err
-             at ./cargo/registry/src/github.com-1ecc6299db9ec823/anyhow-1.0.51/src/lib.rs:659:13
+   0: anyhow::__private::format_err
+             at /home/brian.gianforcaro/.cargo/registry/src/github.com-1ecc6299db9ec823/anyhow-1.0.68/src/lib.rs:666:13
    1: <sum_sq::sum_squares::SumSquares as martian::stage::MartianStage>::main
-             at ./martian-lab/examples/sum_sq/src/sum_squares.rs:94:24
+             at /home/brian.gianforcaro/pub-src/martian-rust/martian-lab/examples/sum_sq/src/sum_squares.rs:94:24
```

Comment:
We don't actually want to have our home directories in this snapshot file.  I thought there was code somewhere that would transform the paths before comparing snapshots.